content/client-server-api/modules/typing_notifications.md

```diff
@@ -6,8 +6,8 @@ type: module
 
 Users may wish to be informed when another user is typing in a room.
 This can be achieved using typing notifications. These are ephemeral
-events scoped to a `room_id`. This means they do not form part of the
-[Event Graph](index.html#event-graphs) but still have a `room_id` key.
+events scoped to a room, but they do not form part of the
```

Comment:
ephemeral events inherently don't form part of the event graph, so "but" is misleading here. 

```suggestion
events scoped to a room, so they do not form part of the
```